### PR TITLE
Shipping blurb

### PR DESCRIPTION
**Pull request by CSellers720**
#### Added a shipping blurb with dynamic ship-by date
shipping-blurb

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "axios": "^0.20.0",
     "express": "^4.17.1",
+    "moment": "^2.29.0",
```
Comment (ccali11) [inline package.json:45]: I read the moment documentation, and it looks like it is going to deprecate relatively soon.
Check it out here: [https://momentjs.com/docs/#/-project-status/](url)

Of course it gets the job done, so definitely no need to adjust.

File: client/src/components/ProductOptions/ProductOptions.jsx [hunk 2]
```diff
@@ -102,7 +119,15 @@ const ProductOptions = (props) => {
...
-        <ShippingBlurb>Shipping blurb here</ShippingBlurb>
+        <ShippingBlurb>
+          <TruckIcon src='https://radiant-dearth.s3.us-east-2.amazonaws.com/icons/my-truck.png'/>
+          <div>
+            Free Shipping. Free 30 Day Returns.
+          </div>
+          <div>
+            Order Now for Delivery <br/>by <ShipDate>{moment(date).format('dddd, MMMM Do')}</ShipDate>
+          </div>
+        </ShippingBlurb>
```
Comment (ccali11) [inline client/src/components/ProductOptions/ProductOptions.jsx]: Looks good!

File: client/src/components/ProductOptions/components/ShippingBlurb.jsx [hunk 3]
```diff
@@ -27,12 +27,9 @@ const ShippingBlurb = () => {
   return (
     <ShippingBlurbDiv>
           <TruckIcon src='https://radiant-dearth.s3.us-east-2.amazonaws.com/icons/my-truck.png'/>
-          <div>
-            Free Shipping. Free 30 Day Returns.
-          </div>
-          <div>
-            Order Now for Delivery <br/>by <ShipDate>{moment(date).format('dddd, MMMM Do')}</ShipDate>
-          </div>
+            Free Shipping. Free 30 Day Returns.<br/>
```
Comment (ccali11) [inline client/src/components/ProductOptions/components/ShippingBlurb.jsx:30]: 👍

**Review by ccali11 — COMMENTED** [review]
Solid.

File: package.json (hunk 1 above)
Comment (CSellers720) [reply package.json:45]: ## Good catch!
I new they were in maintenance mode, but didn't think it was a huge deal. Looking over this more closely, it seems like there are some lighter-weight and more modern solutions. Going to leave this alone for now, but I've got a few alternatives listed below.

moment-alternatives

1. [dayjs](https://www.npmjs.com/package/dayjs)
2. [day-fns](https://www.npmjs.com/package/date-fns)
3. [luxon](https://www.npmjs.com/package/luxon)

**Comment by CSellers720** [comment]
Merge in CLI.

**Comment by CSellers720** [comment]
Merge in CLI

**Pull request closed without merging**